packages/headless/src/index.ts

```diff
@@ -81,3 +82,4 @@ export {NumericFacetValue} from './features/facets/range-facets/numeric-facet-se
 export {RangeFacetSortCriterion} from './features/facets/range-facets/generic/interfaces/request';
 export {buildSearchParameterSerializer} from './features/search-parameters/search-parameter-serializer';
 export * as HighlightUtils from './utils/highlight';
+export {HighlightKeyword} from './utils/highlight';
```

Comment:
The `HighlightKeyword` interface is exported by `HighlightUtils`, but the tooling does extract namespace members.
I exported the interface directly so that the tooling can pick it up.